### DISCUSSION TITLE
Linux Port

### DISCUSSION THREAD

**Discussion by Fcatilizer** [seerge/g-helper#3448] [Ideas]
It might seem like one of the request from another linux user but I was genuinely asking if this g-helper can be ported to linux as well. I've seen a lot of users trying to find a correct way to manage their laptops core features instead of relying on windows software. A lot of linux community also usages asus laptops.

There is already an 'asusctl' package but they are not perfect. If not possible, it's alright. I just thought this application could've been far better than anything present here.

**Comment by seerge** [comment]
@Fcatilizer hello,

I was answering this few times already :) 

**asusctl** from https://asus-linux.org/ is pretty much a "g-helper" for Linux already. It gives you access to all core "firmware-level" settings, same as G-Helper. The rest is probably just windows-specific and can't be moved even. I don't see any point to compete here.
